pkg/services/session/session.go

```diff
@@ -7,36 +7,46 @@ import (
 	"io"
 	"net/http"
 	"net/url"
+	"time"
 
 	"github.com/MicahParks/keyfunc"
-	"github.com/corbado/corbado-go/pkg/logger"
-	"github.com/corbado/corbado-go/pkg/sdk/assert"
-	"github.com/corbado/corbado-go/pkg/sdk/config"
-	"github.com/corbado/corbado-go/pkg/sdk/entity"
-	"github.com/corbado/corbado-go/pkg/sdk/entity/api"
-	"github.com/corbado/corbado-go/pkg/sdk/servererror"
 	"github.com/golang-jwt/jwt/v4"
 	"github.com/pkg/errors"
+
+	"github.com/corbado/corbado-go/pkg/assert"
+	"github.com/corbado/corbado-go/pkg/entities"
+	"github.com/corbado/corbado-go/pkg/generated/api"
+	"github.com/corbado/corbado-go/pkg/logger"
+	"github.com/corbado/corbado-go/pkg/servererror"
 )
 
 type Session interface {
-	ValidateShortSessionValue(shortSession string) (*entity.User, error)
-	GetCurrentUser(shortSession string) (*entity.User, error)
+	ValidateShortSessionValue(shortSession string) (*entities.User, error)
+	GetCurrentUser(shortSession string) (*entities.User, error)
 	ConfigGet(ctx context.Context, params *api.SessionConfigGetParams, editors ...api.RequestEditorFn) (*api.SessionConfigGetRsp, error)
 	LongSessionRevoke(ctx context.Context, sessionID string, req api.LongSessionRevokeReq, editors ...api.RequestEditorFn) error
 	LongSessionGet(ctx context.Context, sessionID string, editors ...api.RequestEditorFn) (*api.LongSessionGetRsp, error)
 }
 
 type Impl struct {
 	client *api.ClientWithResponses
-	config *config.Config
+	config *Config
 	jwks   *keyfunc.JWKS
 }
 
+type Config struct {
```

Comment:
this should either be renamed to `Configuration` too, or we should use `Config` everywhere?

Reply:
stays Config now